kommunicate/src/main/java/com/applozic/mobicomkit/api/notification/NotificationService.java

```diff
@@ -71,6 +72,8 @@ public class NotificationService {
     private String notificationFilePath;
     public static final String BADGE_COUNT = "BADGE_COUNT";
     public static final String NO_ALERT = "NO_ALERT";
+    private final String notificationIconColor;
+    private final String DEFAULT_NOTIFICATION_ICON_COLOR = "3F51B5";
```

Comment:
Don't give a default value. 
If Metadata is not present then don't set the color.